pkgs/development/libraries/tiledb/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, zlib
+, lz4
+, bzip2
+, zstd
+, spdlog_0
+, tbb
+, openssl
+, boost
+, libpqxx
+, clang-tools
+, catch2
+, python
+, gtest
+}:
+
+stdenv.mkDerivation {
+  name = "tiledb";
+  version = "1.6.2";
+
+  src = fetchFromGitHub {
+    owner = "TileDB-Inc";
+    repo = "TileDB";
+    rev = "0eb1eba7a9ff0ada0165bd61c84239bec9cf0f00";
```

allow for bot to update this package
```suggestion
    rev = version;
```